src/module_wrap.cc

```diff
@@ -7,6 +7,7 @@
 #include "util-inl.h"
 #include "node_contextify.h"
 #include "node_watchdog.h"
+#include "node_options.h"
```

Comment:
Probably need to include `node_process.h`?

Reply:
Whoops. Good catch, thanks!

Now I'm trying to figure out how to make experimental warnings emitted only once in C++. And also, add related test cases.

After all the staff done I'll `at` you to review again.